nginx/default.conf

```diff
@@ -21,6 +21,14 @@ server {
   location / {
       root /usr/share/nginx/html;
       try_files $uri /index.html;
+
+      # cache everything for 1 week
+      add_header 'Cache-Control' 'max-age: 604800, immutable' always; 
```

Comment:
Considering all assets should be revisioned, I think a year or even longer would be fine as well. In case we do serve non-revved assets through this setu, we might want to check why?